samza-core/src/main/java/org/apache/samza/coordinator/stream/CoordinatorStreamManager.java

```diff
@@ -129,69 +119,19 @@ public void bootstrapCoordinatorStreamConsumer() {
    */
   public Set<CoordinatorStreamMessage> getBootstrappedStream(String source) {
     if (coordinatorStreamConsumer == null) {
-      throw new UnsupportedOperationException(String.format("CoordinatorStreamConsumer is not initialized in the CoordinatorStreamManager. "
-          + "input source: %s", source));
+      throw new UnsupportedOperationException(String.format("CoordinatorStreamConsumer is not initialized in the CoordinatorStreamManager. "));
```

Comment:
IllegalStateException is more conventional in this scenario.

Reply:
Good catch. thanks.